src/core/extevhan.h

```diff
@@ -18,10 +18,10 @@
 class CEventSourceFB;
 class CFunctionBlock;
 
-#define DECLARE_HANDLER(TypeName)                             \
-  public:                                                     \
-    static const size_t mHandlerIdentifier;              \
-    virtual size_t getIdentifier() const;               \
+#define DECLARE_HANDLER(TypeName)                          \
+  public:                                                  \
+    static const size_t mHandlerIdentifier;                \
+    virtual size_t getIdentifier() const override;         \
```

Comment:
if you add the override you can remove the virtual. Which is cleaner.

Reply:
Fixed.